### PR TITLE
Weather feature

### PR DESCRIPTION
**Pull request by jstloyal**
- Type any valid city and hit enter
- Click the toggle icon on the top right corner of the page to switch from Celsius to Fahrenheit.
- Live link: https://rawcdn.githack.com/jstloyal/Weather-App/f90bdb424182c9d9e7e4159cba9fe3c7165b3dd7/dist/index.html
weather

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parserOptions": {
```
Comment (addod19) [inline .eslintrc.json:8]: Let us add `"parser": "babel-eslint"` in line 7

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @jstloyal, congratulations on the project :fireworks: The project works great :+1: Code is clean :smile: However, before we can approve the project we need to address the comment below:

- Do not make changes in the config files, let us add `"parser": "babel-eslint",` to our `eslintrc.json` file. Kindly visit this [link](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)

Once this is fixed, kindly submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: .eslintrc.json (hunk 1 above)
Comment (jstloyal) [reply .eslintrc.json:8]: After adding that line, I got errors: Failed to load parser 'babel-eslint' declared in '.eslintrc.json': Cannot find module 'babel-eslint'

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hello:wave:**
Is there any particular reason why you are removing `"parser": "babel-eslint"` from .eslintrc.json?
Please, don't modify the config as stated in the previous review.

_**Kindly make this change and submit another code review request.**_

**Cheers:stars:**

### Status: Changes Requested:exclamation:

**Comment by jstloyal** [comment]
> **Hellowave**
> Is there any particular reason why you are removing `"parser": "babel-eslint"` from .eslintrc.json?
> Please, don't modify the config as stated in the previous review.
> 
> _**Kindly make this change and submit another code review request.**_
> 
> **Cheersstars**
> 
> ### Status: Changes Requested
@mosesogwo,
Thank you but it returns and error: Failed to load parser 'babel-eslint' declared in '.eslintrc.json': Cannot find module 'babel-eslint'

**Comment by jstloyal** [comment]
- After adding "parser": "babel-eslint" in line 7, got a recurring error that none of the previous reviewers fail to address after contacting them.
lint-error

**Comment by mosesogwo** [comment]
@jstloyal Have you tried installing babel-eslint manually? `npm i -D babel-eslint`

**Review by MarioDena — APPROVED** [review]
Hello Great work on your project! 

This only needed a simple fix but now the project is ready!

Project Approved!

Cheers! 🎖️

**Pull request merged**